### DISCUSSION TITLE
Save audio stream to an MP3 file

### DISCUSSION THREAD

**Discussion by howardtu2u** [smplayer-dev/smplayer#435] [Q&A]
Hi,

I'm a new smplayer user. Does the application support saving an audio stream like a podcast to a local MP3 file to be played on another device?

Thank You,
Howard

**Comment by smplayer-dev** [comment]
There isn't an option for that but you can use mpv:
https://openwritings.net/pg/mpv/extract-audio-video-file-using-mpv
